### DISCUSSION TITLE
Thieves' Guild and Oracle screens, align "Best Monster" creature sprites vertically to the bottom of the creature sprite rather than to the top of the creature sprite

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4552] [Ideas]
For these screens, I suggest to align the "Best Monster" creature sprites vertically to the bottom of the creature sprite rather than to the top of the creature sprite.
Or eventually centered vertically.

In fheroes2:

![Best monster alignment 001](https://user-images.githubusercontent.com/43583994/140337658-b4721d63-fca3-4375-a320-1bc353b03ee3.png)

![Best monster alignment 002](https://user-images.githubusercontent.com/43583994/140337671-5c651696-095d-495f-9767-210c9908ad38.png)

**Comment by zenseii** [comment]
I agree with this. It looks pretty weird right now. Aligned to the bottom should be best.

**Reply by LeHerosInconnu** [reply]
Hello @zenseii,

> I agree with this. It looks pretty weird right now. Aligned to the bottom should be best.

The flying pig. :rofl:

**Reply by zenseii** [reply]
Archpig 🤣

**Comment by LeHerosInconnu** [comment]
The position has been changed, but I think alignment in relation to the bottom of the sprite would be better visually.

In fheroes2 (1.1.10 9995):

<img width="656" height="518" alt="Best creature position 01" src="https://github.com/user-attachments/assets/66be0680-c653-4c03-9eaa-294cd643b2cb" />

**Comment by LeHerosInconnu** [comment]
This is planned in my mod: https://github.com/ihhub/fheroes2/discussions/10131.
I close the discussion.
